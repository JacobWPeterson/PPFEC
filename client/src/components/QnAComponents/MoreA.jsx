import React from 'react';


class MoreA extends React.Component {
  constructor(props) {
    super(props)
    this.state ={

    }
    this.handleClick = this.handleClick.bind(this)
  }

  handleClick() {
    console.log('clicked');
    this.props.func();
  }


  render () {
    return(
      <div>
        {this.props.display ? <button onClick={this.handleClick}>More Answers</button>: null}
      </div>
    )
  }

}
export default MoreA;