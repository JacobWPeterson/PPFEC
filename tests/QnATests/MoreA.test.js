import React from 'react';
import { mount } from 'enzyme';
import MoreA from '../../client/src/components/QnAComponents/MoreA.jsx';

describe('This is for the More Answers Component', () => {

  const moreAComp = mount(<MoreA />);

  it('Should render More Answers Properly', () => {
    expect(moreAComp).toMatchSnapshot();
  });
});
