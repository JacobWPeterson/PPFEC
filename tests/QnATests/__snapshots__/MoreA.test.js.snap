// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is for the More Answers Component Should render More Answers Properly 1`] = `
<MoreA
  buttonDisplay={true}
>
  <div />
</MoreA>
`;
